packages/bun/src/integrations/bunserver.ts

```diff
@@ -0,0 +1,143 @@
+import { captureException, getCurrentHub, runWithAsyncContext, startSpan, Transaction } from '@sentry/core';
+import type { Integration } from '@sentry/types';
+import {
+  addExceptionMechanism,
+  getSanitizedUrlString,
+  objectify,
+  parseUrl,
+  tracingContextFromHeaders,
+} from '@sentry/utils';
+
+function sendErrorToSentry(e: unknown): unknown {
+  // In case we have a primitive, wrap it in the equivalent wrapper class (string -> String, etc.) so that we can
+  // store a seen flag on it.
+  const objectifiedErr = objectify(e);
```

Comment:
m: Doesn't captureException already do this?

Reply:
I copied what we were doing in SvelteKit, but I think our event processing pipeline does do this. Let me adjust.